Specs/DateTime/English/DateTimeModel.json

```diff
@@ -21171,6 +21171,118 @@
       }
     ]
   },
+  {
+    "Input": "I want to book a flight for the weekend of halloween",
+    "Context": {
+        "ReferenceDateTime": "2021-11-09T18:00:00"
+    },
+    "NotSupported": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "the weekend of halloween",
+        "Start": 28,
+        "End": 51,
+        "TypeName": "datetimeV2.daterange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2021-W43-WE",
+              "type": "daterange",
+              "start": "2021-10-30",
+              "end": "2021-11-01"
+            },
+            {
+              "timex": "2022-W43-WE",
```

Comment:
Is it really the same week number? Just double checking.

Reply:
Yea, I just double checked https://www.epochconverter.com/weeks/2021 (and the 2022 page), and it looks correct to me.